README.md

```diff
@@ -0,0 +1,82 @@
+![Wagtail](https://releases.wagtail.io/wagtail-github-header.png?x)
+
+Wagtail is an open source content management system built on Django, with a strong community and commercial support. It's focused on user experience, and offers precise control for designers and developers. 
+
+![Wagtail screenshot](https://releases.wagtail.io/wagtail-screenshot-with-browser.png)
+
+### Features
+
+* A fast, attractive interface for authors
+* Complete control over design and front-end
+* Scales to millions of pages and thousands of editors
+* Fast out of the box, cache-friendly when you need it
+* Content API for 'headless' sites with de-coupled front-end
+* Runs on a Raspberry Pi or a multi-datacenter cloud platform 
+* StreamField encourages flexible content without compromising structure
+* Powerful, integrated search, using Elasticsearch or PostgreSQL
+* Excellent support for images and embedded content
+* Multi-site and multi-language ready
+* Embraces and extends Django
+
+Find out more at [wagtail.io](https://wagtail.io/).
+
+### Getting started
+
+Wagtail works with Python 3, on any platform.
+
+```
+    pip install wagtail
+    wagtail start mysite
+    cd mysite
+    python manage.py migrate
+    python manage.py createsuperuser
+    python manage.py runserver
+```
+
+For detailed installation and setup docs, see [docs.wagtail.io](http://docs.wagtail.io/).
+
+### Who’s using it?
+
+Wagtail is used by NASA, Google, Oxfam, the NHS, Mozilla, MIT, the Red Cross, Salesforce, NBC, BMW, the US and UK governments. Add your own Wagtail site to [madewithwagtail.org](http://madewithwagtail.org).
+
+### Documentation
+
+[docs.wagtail.io](http://docs.wagtail.io/) is the full reference for Wagtail, and includes guides for developers, designers and editors, alongside release notes and our roadmap.
+
+### Compatibility
+
+Wagtail supports Django 1.11.x and 2.0 on Python 3.4, 3.5 and 3.6. Supported database backends are PostgreSQL, MySQL and SQLite.
```

Comment:
Maybe make bullets out of the Django, Python and database support? That would increase readability for developers and executives who are instantly looking for these kind of characteristics.

Reply:
I wonder if it's worth adding a line in here about older versions supporting Python 2.7, or potentially adding a new docs page or section about this. Django has [a nice writeup](https://docs.djangoproject.com/en/2.0/faq/install/#faq-python-version-support) about Python version support, whereas the Wagtail docs [just say](http://docs.wagtail.io/en/v2.0/getting_started/index.html) "Python" right now without any details.

I can see this line potentially turning away people who (for whatever reason) are stuck on Python 2, but might still like to play with Wagtail, or use 1.13 until such time as they can upgrade to 2.0.